articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,22 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
+- IPv6 clusters
+- [Service Principals](./kubernetes-service-principal.md)
+> [!NOTE]
```

Comment:
Looks like the indentation isn't consistent between line 72 and 73. Both lines should start three spaces in to ensure the note block is indented under line 71 and aligns correctly.

Reply:
Also end the sentence with a period.